cyipopt/superceded/old_ipopt_wrapper.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
```

Comment:
Not clear why this is moved into a `superceded` directory. What's the plan here?

Reply:
I wasn't sure this was being used anywhere as we have `cyipopt/cython/ipopt_wrapper.pyx` and so moved it to `cyipopt/superceded/old_ipopt_wrapper.py` as a reminder that it could be deleted. Do you see any need for keeping it or can I remove this file (and the whole `superseded/` directory) completely for 1.0.0?

Reply:
I think we want to leave this in place and functional. This fork of cyipopt added the scipy-like minimize function. Both things in this file have been public api for some years now, so I'd like to keep it as it was unless we agree to deprecate. I personally think it is a nice addition that is worth maintaining.

Reply:
My bad, I hadn't realised the `minimize()` function was a thing. I'll refactor this stuff back to the way it was.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/10bd4abc247b16f19d910695002491be45cce71a.